tests/libnet/BUILD.bazel

```diff
@@ -3,7 +3,6 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
```

Comment:
commit message: `tests, libnet: Create a dedicated package for dns`

can you add why?

Reply:
Well, it is hard to argue with adding the reasoning, but sometimes it feels really redundant.

Yo leave me no choice now.

Reply:
I truly do not understand your motivation. Is it related to removal of dependencies?